resources/views/admin/databases/index.blade.php

```diff
@@ -36,7 +36,7 @@
         <li><a href="{{ route('admin.databases.new') }}"><i class="fa fa-plus"></i></a></li>
     </ul>
     <div class="tab-content">
-        <div class="tab-pane active" id="tab_databases">
+        <div class="tab-pane {{ Request::input('tab') == 'tab_dbservers' ? '' : 'active' }}" id="tab_databases">
```

Comment:
Not really, if you change it to `tab_databases` no tab will be displayed on page load, that's why I inverted the logic (only show this if we're not on `tab_dbservers` and show the other if we're on that specific one) :)